classes/author.rb

```diff
@@ -0,0 +1,27 @@
+class Author
+  attr_reader :id, :items
+  attr_accessor :first_name, :last_name
+
+  def initialize(params)
+    @id = params[:id] || Random.rand(1..100_000)
+    @first_name = params[:first_name]
+    @last_name = params[:last_name]
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item unless @items.include?(item)
+  end
```

Comment:
The requirements say:
> - add_item method in the Author class
>   - should add self as a property of the item object (by using the correct setter from the item object)

the `Author` uses the `add_item` method to set the `Item` instance into `items` collection, I think it would be better to have an `add_author` setter method in `Item` class in order to follow the requirements, and set the `author` property for `Item` as read only. Because you can do something like `item.author = 1` and `1` is not an `Item` instance.